cpp/src/io/parquet/page_decode.cuh

```diff
@@ -21,10 +21,21 @@
 
 #include <io/utilities/block_utils.cuh>
 
+#include <cuda/atomic>
 #include <cuda/std/tuple>
 
 namespace cudf::io::parquet::gpu {
 
+enum class decode_error {
```

Comment:
Should this go in `parquet_gpu.hpp`? I'm thinking ahead to using the same codes in `gpuDecodePageHeaders` to detect the error from #13656.

Reply:
Yeah, you're right.
Tangential to the suggestion: I think current check already cover #13656. Will verify in a bit.

Reply:
I mixed it up with another "should fail instead of returning garbage" issue. No current checks catch #13656.